frontend/src/index.js

```diff
@@ -3,9 +3,19 @@ import ReactDOM from 'react-dom';
 import { BrowserRouter } from 'react-router-dom';
 import { Main } from './layout';
 import ContextProvider from './provider/ContextProvider';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { fas } from '@fortawesome/free-solid-svg-icons';
+import {
+	faStar,
+	faMapPin,
+	faPhone,
+	faEnvelope,
+} from '@fortawesome/free-solid-svg-icons';
```

Comment:
Duplicate of @fortawesome/free-solid-svg-icons

Reply:
Done!